pyfr/solvers/baseadvecdiff/system.py

```diff
@@ -20,6 +22,10 @@ def rhs(self, t, uinbank, foutbank):
         q1 << kernels['bcint', 'con_u']()
         q1 << kernels['eles', 'tgradpcoru_upts']()
 
+        q2 << kernels['mpiint', 'avis_fpts_send']()
+        q2 << kernels['mpiint', 'avis_fpts_recv']()
+        q2 << kernels['mpiint', 'avis_fpts_unpack']()
```

Comment:
This order will impact our ability to overlap compute with communication.  Reorder so the two unpacks are at the end.